qiskit/pulse/__init__.py

```diff
@@ -13,56 +13,77 @@
 # that they have been altered from the originals.
 
 """
-===============================
-OpenPulse (:mod:`qiskit.pulse`)
-===============================
-
 .. currentmodule:: qiskit.pulse
 
-Channels
-========
+===========================
+Pulse (:mod:`qiskit.pulse`)
+===========================
 
-.. autosummary::
-   :toctree: ../stubs/
+Qiskit-Pulse is a pulse-level quantum programming kit. This lower level of programming offers the
+user more control than programming with :py:class:`~qiskit.circuit.QuantumCircuit` s.
 
-   DriveChannel
-   MeasureChannel
-   AcquireChannel
-   ControlChannel
-   RegisterSlot
-   MemorySlot
+Extracting the greatest performance out from quantum hardware requires real-time pulse-level
```

Comment:
I see why 'out' is in there but I think it flows better without it.

```suggestion
Extracting the greatest performance from quantum hardware requires real-time pulse-level
```

Reply:
agree